phpcr-shell/querying.rst

```diff
@@ -196,7 +196,7 @@ Set the value of ``a.title`` to the value of the property :
 
 .. code-block:: bash
 
-    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getProperty('some_property')')
+    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getPropertyValue("some_property")')
```

Comment:
Yeah, I think in this case `getPropertyValueWithDefault("some_property", null)` would probably be best.